src/main/java/teammates/ui/controller/AdminInstructorAccountAddAction.java

```diff
@@ -297,37 +298,24 @@ private String getDemoCourseIdRoot(String instructorEmail){
     private String generateNextDemoCourseId(String instructorEmailOrProposedCourseId, int maximumIdLength){
         final boolean isFirstCourseId = instructorEmailOrProposedCourseId.contains("@");
         if(isFirstCourseId){
-            return trimCourseIdToMaximumLengthIfNecessary(getDemoCourseIdRoot(instructorEmailOrProposedCourseId)
-                    , maximumIdLength);
+            return new StringHelper().truncateHead(getDemoCourseIdRoot(instructorEmailOrProposedCourseId)
```

Comment:
@bamboo3250 you are right , its not necessary to use instance. I made the function truncateHead static.